low-level/sensors/impl/generic/node_hw.py

```diff
@@ -1016,6 +1084,15 @@ def _parse_psu_supply_info(self, index, date, _time, sensor, sensor_num, event,
 
         if is_last:
             specific_info.update(dynamic)
+        
+        if alert_type == "fault":
+            self.faulty_resources[sensor_id] = {
+                'status': status,
+                'event': event,
+                'resource': 'psu_supply'
```

Comment:
Resource name is already defined one. Please match with this.
```
    TYPE_PSU_SUPPLY = 'Power Supply'
```

Reply:
Done